api/src/main/java/org/eclipse/microprofile/metrics/annotation/package-info.java

```diff
@@ -86,5 +86,5 @@
  * </pre>
```

Comment:
need to fix comments on lines 46-48 to remove references to metric types we no longer have